Mathlib/Geometry/Manifold/LocalDiffeomorph.lean

```diff
@@ -269,3 +274,132 @@ noncomputable def IslocalDiffeomorph.diffeomorph_of_bijective
       exact this ▸ (Φ x).map_source hx }
 
 end Basic
+
+section helper -- FIXME: move to Algebra.Module.Basic
+variable {R : Type*} [Ring R]
+variable {E : Type*} [TopologicalSpace E] [AddCommMonoid E] [Module R E]
+variable {F : Type*} [TopologicalSpace F] [AddCommMonoid F] [Module R F]
+
+/-- `g ∘ f = id` as `ContinuousLinearMap`s implies `g ∘ f = id` as functions. -/
+lemma LeftInverse.of_composition {f : E →L[R] F} {g : F →L[R] E}
```

Comment:
I think the right way to avoid this lemma is to make a `ContinuousLinearMap` version of [LinearMap.congr_fun](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Algebra/Module/LinearMap.html#LinearMap.congr_fun)

Reply:
That's better indeed. Good idea!